test-cases-internal/src/main/java/com/palantir/conjure/verification/GenerateServerServices.java

```diff
@@ -43,6 +47,35 @@ public static void main(String... args) throws IOException {
                 generateSingleQueryParamService(testCases.getSingleQueryParam()));
     }
 
+    private static Map<String, Object> generateAutoDeserializeConfirmService(List<BodyTests> body) {
+        ImmutableMap.Builder<String, Object> endpoints = ImmutableMap.builder();
+        endpoints.put("confirm", ImmutableMap.builder()
```

Comment:
Doesn't matter for client-side testing, but our Rust server implementation rejects ambiguous routes.

Reply:
Thanks for flagging. I don't think this will be an issue since the verifier is the only server expected to implement the confirm service